test/06_show.jl

```diff
@@ -80,37 +79,72 @@ end
     ov1 = CategoricalValue(1, opool)
     ov2 = CategoricalValue(2, opool)
     ov3 = CategoricalValue(3, opool)
+    
+    if Base.VERSION >= v"1.4.0-DEV.591"
```

Comment:
Don't worry about Julia master, that change is going to be reverted soon (it's been that way for 1.2 and 1.3). Anyway these changes shouldn't be needed for this PR.